posix.mak

```diff
@@ -728,7 +728,7 @@ endif
 $G/docs-latest.json : ${DMD_LATEST} ${DMD_LATEST_DIR} \
 			${DRUNTIME_LATEST_DIR} ${PHOBOS_LATEST_FILES_GENERATED} | dpl-docs
 	find ${DMD_LATEST_DIR}/src -name '*.d' | \
-		sed -e /mscoff/d -e /objc_glue.d/d ${DMD_EXCLUDE} \
```

Comment:
I think you have hit a chicken-egg problem here.
I guess you have to do sth. like:

```make
// remove me after https://github.com/dlang/dmd/pull/7513 has been merged
ifneq (,$(wildcard $(DMD_DIR)/src/dmd/objc_glue_stubs.d))
   DMD_EXCLUDE_PRERELEASE += -e /objc_glue.d/d 
endif
```

Note that this needs to be a separate variable for each `.json` file as otherwise this will case problems on the next master-stable branchoff.

Reply:
Done. I'm not sure if this is correct.